src/ucp/api/ucp.h

```diff
@@ -1188,6 +1196,18 @@ typedef struct ucp_worker_params {
      */
     uint64_t                flags;
 
+    /**
+     * Worker name is intended for identification of the worker during tracing
```

Comment:
IMHO the first sentence has to be rewritten. @tonycurtis  I don't like the passive form. `The worker name is an id of the worker object for the tracing tool`?

Reply:
Tracing and analysis tools can identify the worker using the name.

@shamisp @tonycurtis 
How about that? I tried to make it a little more concise.

Reply:
Maybe "this name", but otherwise, yes.